### ISSUE TITLE
PhpStan(erred) on Notice

### ISSUE THREAD

**Issue by Skarlso** [Rockstar04/SublimeLinter-contrib-phpstan#2]
Hi!

So apparently, when phpstan is using a phpstan.neon file it outputs this:

```bash
SublimeLinter: WARNING: phpstan output:
Note: Using configuration file [...]/docroot/phpstan.neon.
```

Which is picked up as a warning for some reason. It shouldn't.

For the record. There is no other error / notice in the log regarding to phpstan. So I'm guessing it doesn't wait for any kind of output in case everything is okay.

**Comment by ryanpcmcquen** [comment]
@Rockstar04, any progress on this?

**Comment by Rockstar04** [comment]
Not really... I apparently incorrectly assumed that the regex would filter these messages out. I tested the regex against this warning with RegExr and it filters it correctly.

**Comment by ryanpcmcquen** [comment]
Yeah, it doesn't apply to console output.

**Comment by ryanpcmcquen** [comment]
@Rockstar04, have you tried redirecting `stderr`?

For instance, finishing the command with this?
```2>/dev/null```

**Comment by ryanpcmcquen** [comment]
@Skarlso, I have a fix for this in #9!

**Comment by Skarlso** [comment]
Fantastic! Thank you very much!
